doc/api/pyplot_summary.rst

```diff
@@ -1,8 +1,41 @@
-Plotting commands summary
-=========================
+Below we describe several common approaches to plotting with Matplotlib.
+
+.. contents::
+
+The Pyplot API
+--------------
+
+The :mod:`matplotlib.pyplot` module contains functions that allow you to generate
+many kinds of plots quickly. For examples that showcase the use
+of the :mod:`matplotlib.pyplot` module, see the
+:ref:`sphx_glr_tutorials_01_introductory_pyplot.py`
+or the :ref:`pyplots_examples`. We also recommend that you look into
+the object-oriented approach to plotting, described below.
 
 .. currentmodule:: matplotlib.pyplot
 
 .. autofunction:: plotting
 
+The Object-Oriented API
```

Comment:
ack yeah I'm sorry - I've got a buncha PRs open so it's hard to remember which one I've pushed to the website. I imagined just adding the section on OO stuff to the pyplot summary. It felt weird to put it at the beginning because the page is called `pyplot`. What if we instead called the page `plotting API` instead of `pyplot`? Then I could add the OO stuff first.